_test/test_utilities_herbert/test_object_lookup_2.m

```diff
@@ -0,0 +1,760 @@
+classdef test_object_lookup_2 < TestCaseWithSave
```

Comment:
Again, none of these methods are compared with save

Reply:
See earlier reply